### PR TITLE
Provision ESXi with MAAS

### PR DESCRIPTION
**Pull request by roaksoax**
Add new tutorial: Provisiong ESXi with MAAS

### CODE REVIEW

File: tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+## Overview
+Duration: 0:05
+
+In this tutorial, you will learn how to create MAAS images of ESXi using MAAS provided packer scripts.
+
+ESXi support was first introduced in MAAS 2.5, but expanded in 2.6 allowing for the configuration of VMFS datastores, registration to vCenter and others. For more information about what features MAAS support, please refer to the following blog post <Provisioning ESXi with MAAS: An Overview> ..
+
+In order to create ESXi images MAAS leverages the use of packer, and alongside the image creation process, it introduces a series of scripts and dependencies that makes it possible for MAAS to customize each ESXi deployment, including network configuration, datastore creation and user level customization.
+
+### What you'll learn
```
Comment (pmahnke) [inline tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md]:
```suggestion
### You will learn how to:
```

File: tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+---
+
+# Provision ESXi with MAAS
+
+## Overview
+Duration: 0:05
+
+In this tutorial, you will learn how to create MAAS images of ESXi using MAAS provided packer scripts.
+
+ESXi support was first introduced in MAAS 2.5, but expanded in 2.6 allowing for the configuration of VMFS datastores, registration to vCenter and others. For more information about what features MAAS support, please refer to the following blog post <Provisioning ESXi with MAAS: An Overview> ..
```
Comment (pmahnke) [inline tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md]:
```suggestion
ESXi support was first introduced in MAAS 2.5, but expanded in 2.6 allowing for the configuration of VMFS datastores, registration to vCenter and others. For more information about what features MAAS support, please refer to the following blog post [Provisioning ESXi with MAAS: An Overview](https://ubuntu.com/blog/provisioning-esxi-with-maas).
```

File: tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md (hunk 1 above)
Comment (caldav) [reply tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md]: "What you'll learn" generates a custom styling (:heavy_check_mark:) for the bullet list. Let's keep it consistent with "most" other tutorials.

Comment (roaksoax) [reply tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md]: I don't follow. That's exactly the styling that's been generated.

**Pull request merged**